amundsenatlastypes/schema/01_1_column_schema.json

```diff
@@ -1,9 +1,59 @@
 {
+  "structDefs": [
+    {
+      "name": "StatisticsEntry",
+      "attributeDefs": [
+        {
+          "name": "stat_name",
+          "typeName": "string",
+          "isOptional": false,
+          "cardinality": "SINGLE",
+          "isUnique": false,
+          "isIndexable": false
+        },
+        {
+          "name": "stat_val",
+          "typeName": "double",
```

Comment:
maybe it's good time to consider having this field in different type than double ? in the future we will have other kinds of statistics (like histograms etc). II propose having stat_val of `string` type and maybe additional field stat_kind which is optional but can contain info about type of stat (like double, string, histogram)

Reply:
out of the scope of this change